Google.Api.Gax.Grpc.IntegrationTests/ClientBuilderBaseTest.cs

```diff
@@ -226,6 +227,58 @@ public async Task TokenAccessMethod()
             await ValidateResultAsync(builder, validator);
         }
 
+        [Fact]
+        public async Task JwtClientEnabledTest()
+        {
+            var builder = new SampleClientBuilder(true, false) { JsonCredentials = DummyServiceAccountCredentialFileContents };
+            ChannelBase channelFromPool = builder.ChannelPool.GetChannel(GrpcCoreAdapter.Instance, SampleClientBuilder.DefaultEndpoint, SampleClientBuilder.DefaultOptions);
+
+            // Jwt enabled for client and disabled for pool
+            // We won't use channel pool
```

Comment:
```suggestion
        [Theory]
        [InlineData(true, false)]
        [InlineData(false, true)]
        public async Task DoesNotUsePool(bool clientUsesJwt, bool poolUsesJwt)
        {
            var builder = new SampleClientBuilder(clientUsesJwt, poolUsesJwt) { JsonCredentials = DummyServiceAccountCredentialFileContents };
            ChannelBase channelFromPool = builder.ChannelPool.GetChannel(GrpcCoreAdapter.Instance, SampleClientBuilder.DefaultEndpoint, SampleClientBuilder.DefaultOptions);

            // We won't use channel pool because it's either just
            // enabled for the client or just enabled for the pool.
```

Reply:
applied, added parameterless